src/InputForm.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+export default function InputForm({ handleChange, formData }) {
+  return (
+    <div>
+      <form>
+        <div>
+          <div style={{ fontFamily: "Sigmar One, sans-serif" }}>
```

Comment:
It is interesting to see styles injected inline into JSX like this

Reply:
This is a remnant of the previous voyage.